tcms/rpc/tests/test_testcase.py

```diff
@@ -3,26 +3,86 @@
 
 from xmlrpc.client import Fault, ProtocolError
 
+from attachments.models import Attachment
 from django.contrib.auth.models import Permission
+from django.core.exceptions import ValidationError
+from django.test import TestCase as DjangoBaseTestCase
 from tcms_api import xmlrpc
 
 from tcms.core.helpers import comments
 from tcms.management.models import Priority
+from tcms.rpc.api.testcase import _validate_cc_list
 from tcms.rpc.tests.utils import APITestCase, APIPermissionsTestCase
+from tcms.rpc.utils import add_attachment
 from tcms.testcases.models import Category, TestCase, TestCaseStatus
 from tcms.tests import remove_perm_from_user
 from tcms.tests.factories import (CategoryFactory, ComponentFactory,
                                   ProductFactory, TagFactory, TestCaseFactory,
                                   TestPlanFactory, UserFactory, VersionFactory)
 
 
-class TestNotificationRemoveCC(APITestCase):
-    """ Tests the XML-RPC testcase.notication_remove_cc method """
+class TestValidateEmail(DjangoBaseTestCase):
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.default_cc = 'example@example.com'
+
+    def test_non_list_email_collection(self):
+        with self.assertRaisesRegex(TypeError, 'cc_list should be a list object.'):
+            _validate_cc_list((self.default_cc,))
```

Comment:
You can pass a hard-coded string value here, like you do below and will not need the setup method at all.